cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -378,6 +378,7 @@ func NewControllerInitializers(loopMode ControllerLoopMode) map[string]InitFunc
 	controllers["clusterrole-aggregation"] = startClusterRoleAggregrationController
 	controllers["pvc-protection"] = startPVCProtectionController
 	controllers["pv-protection"] = startPVProtectionController
+	controllers["ttl-after-finished-controller"] = startTTLAfterFinishedController
```

Comment:
while this is alpha, ensure it is in the ControllersDisabledByDefault list

Reply:
`startTTLAfterFinishedController` won't be started if TTLAfterFinished feature is disabled.

Reply:
ah, missed that. that's fine.